### ISSUE TITLE
Docs: Updates to kubernetes with cert-manager install guide

### ISSUE THREAD

**Issue by charliesolomon** [openfaas/faas#1570]
## My actions before raising this issue
- [X] Followed the [troubleshooting guide](https://docs.openfaas.com/deployment/troubleshooting/)
- [X] Read/searched [the docs](https://docs.openfaas.com/)
- [X] Searched [past issues](/issues)

The install guide for openfaas on kubernetes with cert-manager could be refreshed.
Docs published here: https://docs.openfaas.com/reference/ssl/kubernetes-with-cert-manager/

## Expected Behaviour
Commands in docs can be executed without errors and openfaas works as expected when finished.

## Current Behaviour
- Helm v3.x does not use Tiller, could lead to confusion
- "Install OpenFaas" link comes before ingress set up in docs, but OpenFaas install guide deployment command using ingress assumes ingress is already deployed
- Deployment command for nginx ingress throws error ("unknown flag: --name")

## Possible Solution
- Update references to helm/tiller to clarify helm v3.x use without tiller
- Fix docs by installing nginx ingress before deploying openfaas
- stable/nginx-ingress helm chart is deprecated, update using ingress-nginx docs: https://kubernetes.github.io/ingress-nginx/deploy/
- add option to use arkade commands?

## Steps to Reproduce (for bugs)
See below for environment in which I tested these docs.

## Context
Setting up OpenFaas with TLS on DigitalOcean k8s to test integration with Keycloak SSO.

## Your Environment

* FaaS-CLI version ( Full output from: `faas-cli version` ):
CLI:
 commit:  ea687659ecf14931a29be46c4d2866899d36c282
 version: 0.11.8
Your faas-cli version (0.11.8) may be out of date. Version: 0.12.13 is now available on GitHub.

* Docker version `docker version` (e.g. Docker 17.0.05 ):
N/A

* Are you using Docker Swarm or Kubernetes (FaaS-netes)?
DigitalOcean k8s 1.18

* Operating System and version (e.g. Linux, Windows, MacOS):
MacOS Catalina

**Comment by alexellis** [comment]
I agree that this page could use a couple of minor updates, but this isn't required for the OIDC plugin. The blog post shows you how to deploy all of this using arkade and kubectl. 

Please read the blog post again and use the steps provided for the Keycloak testing.

**Comment by charliesolomon** [comment]
I agree as well Alex.  Not having any problems installing OpenFaas using the blog post you referenced.

I was using this documentation to better understand configuring TLS using cert-manager, so I could apply that to a Keycloak installation... ran across the above potential fixes and decided to open an issue so that I can help with your docs... after my Keycloak experiment :)

**Comment by alexellis** [comment]
@charliesolomon would you be open to raising a PR with the changes to the docs repo?
